canonical-websites/insights.ubuntu.com
# Build two versions of new homepage

## Done

Build new versions of homepage

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8023/](http://0.0.0.0:8023/)
- Check homepage matches [this design](https://user-images.githubusercontent.com/36884067/37915540-a9a2bb12-3111-11e8-82e2-01683913baf1.jpg)
- View the site locally in your web browser at: [http://0.0.0.0:8023/home](http://0.0.0.0:8023/home)
- Check homepage matches [this design](https://user-images.githubusercontent.com/36884067/37915538-a978fd72-3111-11e8-8d07-f0db8580e101.jpg)


## Issue / Card

Fixes #254
